server/jetstream_cluster.go

```diff
@@ -1136,7 +1148,10 @@ func (js *jetStream) processStreamAssignment(sa *streamAssignment) {
 	accStreams[stream] = sa
 	cc.streams[acc.Name] = accStreams
 
-	isMember := sa.Group.isMember(cc.meta.ID())
+	var isMember bool
+	if sa.Group != nil && cc.meta != nil {
```

Comment:
If those are possible legitimate conditions (group and/or meta being nil), a small comment indicating what that means may help the reader.

Reply:
It should not happen, if it does, at least for now, that is an error.